lib/tasks/annotations.rake

```diff
@@ -0,0 +1,25 @@
+namespace :annotations do
+  desc 'Compare Ruby\'s text to what Vue returns'
+  task(:compare_ruby_text_to_vue => :environment) do
+    [Case, TextBlock].each do |klass|
+      i = 0
+      count = klass.count
+      klass.find_each do |inst|
+        i += 1
+        ruby_text = HTMLUtils.parse(inst.content).text
+        vue_text = HTMLUtils.parse(Vue::SSR.render(inst.content)).text
+        diffs = DiffHelpers.get_diffs(ruby_text, vue_text).select { |d| d[0] != :equal }
```

Comment:
Renamed to Differ

Reply:
Fixed in https://github.com/leppert/h2o/commit/d2d6e969bab42fff33c601e9fa45b232cd2af132